.travis.yml

```diff
@@ -6,9 +6,7 @@ os:
   - linux
 
 cache:
-  directories:
-    - node_modules
-
+  npm: false
```

Comment:
It's not a good way for resolve problem, it will make travis CI be slow.

Reply:
yes,we can add this when pr merged. because checks will not pass when use cache.